src/components/Jumpstart.tsx

```diff
@@ -1,18 +1,45 @@
-import React from 'react';
-import { Helmet } from 'react-helmet';
-import { makeStyles } from '@material-ui/core/styles';
-import { Container, Box, Paper } from '@material-ui/core';
-
-/*
-Template for future page components.
-Use MUI's CSS-in-JS makeStyles() as shown below for more complex CSS/
-Otherwise, use <Box> components where possible for inline styling.
-Every page must have a Helmet tag for SEO purposes.
-*/
+import React from "react";
+import { Helmet } from "react-helmet";
+import { makeStyles } from "@material-ui/core/styles";
+import {
+  Container,
+  Box,
+  Paper,
+  Grid,
+  Typography,
+  Button,
+  Accordion,
+  AccordionSummary,
+  AccordionDetails,
+} from "@material-ui/core";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+import Hero from "./Hero";
+import { ReactComponent as JumpstartSvg } from "../svg/jumpstart.svg";
+import JumpstartAccordion from "./JumpstartAccordion";
 
 const useStyles = makeStyles({
-  example: {
-    color: 'red',
+  section: {
+    marginTop: "10vh",
+    marginBottom: "10vh",
+  },
+  subtext: {
+    marginTop: "1.5vh",
+  },
+  flexSection: {
+    marginTop: "10vh",
+    marginBottom: "10vh",
+    display: "flex",
+    alignItems: "center",
+    justifyContent: "center",
+  },
+  button: {
+    width: "40vh",
+    height: "10vh",
+    fontSize: "48px",
```

Comment:
! setting font size is scary because it might not be responsive I'll have to take a look

Reply:
![image](https://user-images.githubusercontent.com/23691775/89490072-c3cc4d80-d760-11ea-9ae6-b9f3b1c6c805.png)
mhm so thats why we shouldnt do that

Reply:
ah yeah good catch, do you know how we could set the underlying text tag instead?